src/vuex/modules/connection.js

```diff
@@ -18,6 +18,16 @@ export default function({ apollo }) {
   }
 
   const actions = {
+    async getNetworkByAddress(store, address) {
+      const { data } = await apollo.query({
```

Comment:
This is not a 1:1 mapping. Usually testnets and mainnets have the same prefix.

Reply:
@jbibla what do you think we should do here? Offer a mainnet / testnet selection?

Reply:
@faboweb I know, it's just for testing logging in with network change.

Reply:
> what do you think we should do here? Offer a mainnet / testnet selection?

no, not a selection for the user. probably, we should store and pass the network id with the address when sending from the extension.